app/Http/Controllers/RelationshipController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use App\Relationship;
+use Illuminate\Http\Request;
+
+class RelationshipController extends Controller
+{
+    public function store()
+    {
+        $attributes = $this->validateRelationship();
+        Relationship::create($attributes);
+        $user = User::whereId($attributes['followed_id'])->first();
+        session()->flash('message', 'You have followed ' . $user->first_name);
+
+        return redirect('/profile/'.$attributes['followed_id']);
```

Comment:
instead of calling session function you can directly make a flash message with `with()` function. e.g

`return redirect(...)->with('message', 'Message Here')`

Reply:
Will resolve this in a bit.

Reply:
This is resolved in fcf53d0